homeassistant/components/qbus/strings.json

```diff
@@ -0,0 +1,19 @@
+{
+  "config": {
+    "flow_title": "Controller {serial}",
+    "step": {
+      "discovery_confirm": {
+        "title": "Add controller",
+        "description": "Add controller {serial}?"
+      }
+    },
+    "abort": {
+      "already_configured": "Controller already configured",
+      "discovery_in_progress": "Discovery in progress",
+      "not_supported": "Configuration for QBUS happens through MQTT discovery"
```

Comment:
Can we be more verbose on what this means? What must an user do if they want to set up and they get here?